imports/plugins/included/payments-paystack/server/methods/paystack-payment-methods.app-test.js

```diff
@@ -0,0 +1,237 @@
+import { Meteor } from "meteor/meteor";
+import { expect } from "meteor/practicalmeteor:chai";
+import { sinon } from "meteor/practicalmeteor:sinon";
+
+import { PaystackApi, RISKY_TEST_CARD } from "./paystackapi";
+
+const paymentMethod = {
+  processor: "Generic",
+  storedCard: "Visa 4242",
+  paymentPackageId: "vrXutd72c2m7Lenqw",
+  paymentSettingsKey: "paystack-paymentmethod",
+  status: "captured",
+  mode: "authorize",
+  createdAt: new Date()
+};
+
+describe("PaystackApi", function () {
+  let sandbox;
+
+  beforeEach(function () {
+    sandbox = sinon.sandbox.create();
+  });
+
+  afterEach(function () {
+    sandbox.restore();
+  });
+
+  it("should return data from ThirdPartyAPI authorize", function () {
+    const cardData = {
+      name: "Test User",
+      email: "test@gmail.com"
+    };
+    const paymentData = {
+      currency: "USD",
+      total: "19.99"
+    };
+
+    const transactionType = "authorize";
+    const transaction = PaystackApi.methods.authorize.call({
+      transactionType: transactionType,
+      cardData: cardData,
+      paymentData: paymentData
+    });
+    expect(transaction).to.not.be.undefined;
```

Comment:
Good job including test suites. However, how can we make this more concrete, as opposed to just testing that  it's defined?

Reply:
Hey @Merdoth could you deal with this. It's a good for a first try. But could you test exactly what should be returned back.